packages/server/src/automations/tests/utilities/AutomationTestBuilder.ts

```diff
@@ -398,21 +398,19 @@ class AutomationBuilder extends BaseStepBuilder {
 
   async run() {
     const automation = await this.save()
-    const results = await testAutomation(
-      this.config,
-      automation,
-      this.triggerOutputs
+    const response = await this.config.api.automation.test(
+      automation._id!,
+      this.triggerOutputs as TestAutomationRequest
```

Comment:
I don't like having to do this cast but `TriggerOutputs` isn't compatible with `TestAutomationRequest`. This is a problem that we should rectify, but I didn't get to it in this PR.